### DISCUSSION TITLE
SINGLE_BLOCK_ENABLE

### DISCUSSION THREAD

**Discussion by JasonTitcomb** [grblHAL/core#422] [Q&A]
Hi All,
I am trying to understand if single block is implemented.
I see SINGLE_BLOCK_ENABLE in my_machine.h but is it implemented?

**Comment by terjeio** [comment]
Not yet - work in progress. Until ready send `$S` from the MDI to toggle single block mode.
BTW which driver do you have?

**Reply by JasonTitcomb** [reply]
BOARD_T41U5XBB

**Reply by terjeio** [reply]
I have just commited a version that should work, also from the [Web Builder](http://svn.io-engineering.com:8080/) - enable in the new _Optional inputs_ tab. Please report back if not.
